packages/cardpay-reward-programs/cardpay_reward_programs/main.py

```diff
@@ -57,7 +57,7 @@ def run_reward_program(
     payment_list = rule.run(
         parameters["run"]["payment_cycle"], parameters["run"]["reward_program_id"]
     )
-    tree = PaymentTree(payment_list.to_dict("records"))
+    tree = PaymentTree(payment_list.to_dict("records"), run_parameters=parameters)
```

Comment:
I am fine with "run_parameters". 

When I think, I tend to refer metadata as something that occurs after a run whereas I tend to refer to parameters as something that occurs before a run. 

Since, "run_parameters" can be considered as something reproducible before a run. I prefer it over something e.g. "run_metadata". It's also clear since you are extracting "run_parameters" from the metadata of parquet file.

Reply:
That distinction makes sense to me, thanks for going through it.